### ISSUE TITLE
Pointerdown is not firing on checkbox input element

### ISSUE THREAD

**Issue by louh** [deltakosh/handjs#10]
If I have a checkbox input, and have attached pointerdown events on it, the pointerdown event does not fire. 

Tested on Chromebook Pixel, Chrome OS version 43.0.2357.134 and on iPad iOS8/Safari 8.

The workaround is to have my script listen for the "change" event instead. (Previously I was using "click" which also worked.)

**Comment by SaschaNaz** [comment]
http://jsfiddle.net/v2afnmjL/

Does this also not work on your Pixel and iPad?

**Comment by louh** [comment]
I'm having a problem with jsfiddle on iPad so ignoring that for now. This
does work on the Chromebook.

**Comment by SaschaNaz** [comment]
Hmm. Does mousedown work on your iPad?

http://jsfiddle.net/j8pnqra2/

**Comment by SaschaNaz** [comment]
How about touchstart? Sorry for continuous questions, but your answer will help.

http://jsfiddle.net/7go5frs3/

**Comment by louh** [comment]
No worries, thanks so much for being responsive. I seem to have a problem with the checkbox where on iPad, the hitbox for the twitter button is somehow floating right over the checkbox, so every time I tap that it opens Twitter in a new window. I'm working around that by putting some <br>s before the label so that I could move it down on the page and be able to test the checkbox properly.

So on iPad, all three (pointerdown, mousedown, touchstart) do work.

I have a much more complicated application, and it's possible that something else could be unpredictably conflicting with it?

**Comment by SaschaNaz** [comment]
I'm not sure about other unpredictable situations but I can say that iframes including Twitter buttons can interrupt others.

Thanks for your answer and explanation :D

**Comment by louh** [comment]
Thanks again for your help.

By the way, in case you're interested in the application we're working on. [Streetmix](http://github.com/codeforamerica/streetmix), which we did back before dual-input touch and mouse machines were common, so we made it listen to click events only, OR touch events only if a touch device was detected. Which basically means a bunch of newer devices found themselves unable to use the app at all on their preferred mode of input! We've recently gotten most of the way on normalizing all the inputs to just listen for pointer events, thanks to Hand.js :) :+1:
